ds-ml-service/files/src/mlservice/services/infrastructure/remote/ssh/executors.py

```diff
@@ -135,13 +135,18 @@ def run_training(self) -> None:
         """
         # remote training with remote dvc repository
         if self._dvc_remote:
-            train_script = """{0}/services/infrastructure/remote/scripts/remote_trainer.py --env {0}
-            --debug {1} --dvc_remote {2} --dvc_user {3} --dvc_password {4}""". \
+            train_script = """{0}/services/infrastructure/remote/scripts/remote_trainer.py \\
+                --env {0} \\
```

Comment:
hi @henninglebbaeus , 
did you tried exactly the same structure but only one backslach?

Reply:
I think it has to be a double backslash, since backslash is an escape character in Python.

Reply:
got it! basically we need to escape it! thanks!